editor/src/clj/editor/bundle_dialog.clj

```diff
@@ -925,12 +953,14 @@
   (run [bundle! prefs presenter stage workspace]
     (save-prefs! presenter prefs)
     (let [bundle-options (get-options presenter)
-          output-prefs-key (str "bundle-output-directory-" (hash (g/node-value workspace :root)))
+          output-prefs-key (prefs/make-project-specific-key "bundle-output-directory" workspace)
```

Comment:
I moved this functionality into `prefs` a while ago to be able to reuse it, so it makes sense to use it here as well